.github/workflows/pull-request.yml

```
@@ -0,0 +1,354 @@
+      - name: Checking for correct formatting of code
+        run: dart format --set-exit-if-changed .
+      - name: setup python
+        uses: actions/setup-python@v5
+      - name: Count lines of code in each file
+        run: chmod +x ./.github/workflows/scripts/countline.py
+      - name: Running count lines
+        run: ./.github/workflows/scripts/countline.py --exclude_directories test/ --exclude_files lib/custom_painters/talawa_logo.dart lib/custom_painters/language_icon.dart lib/custom_painters/whatsapp_logo.dart lib/utils/queries.dart lib/view_model/after_auth_view_models/profile_view_models/profile_page_view_model.dart lib/view_model/pre_auth_view_models/select_organization_view_model.dart lib/views/after_auth_screens/profile/profile_page.dart lib/view_model/main_screen_view_model.dart lib/views/after_auth_screens/events/create_event_page.dart lib/views/after_auth_screens/org_info_screen.dart lib/views/after_auth_screens/events/manage_volunteer_group.dart lib/views/after_auth_screens/events/create_agenda_item_page.dart lib/views/after_auth_screens/events/edit_agenda_item_page.dart lib/utils/event_queries.dart
+      - name: Check for presence of ignore directives corresponding to custom lints
+        run: chmod +x ./.github/workflows/scripts/check_ignore.py
+      - name: Run check_ignore
+        run: |
```

actionlint (1.7.9)

"github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)